src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx

```diff
@@ -791,3 +787,19 @@ export const AutomatedAnalysisCard: React.FunctionComponent<AutomatedAnalysisCar
     </Card>
   );
 };
+
+export const AutomatedAnalysisCardDescriptor: DashboardCardDescriptor = {
+  title: 'Automated Analysis',
+  description: `
+Assess common application performance and configuration issues.
+    `,
+  descriptionFull: `
+Creates a recording and periodically evalutes various common problems in application configuration and performance.
+Results are displayed with scores from 0-100 with colour coding and in groups.
+This card should be unique on a dashboard.
+      `,
+  component: AutomatedAnalysisCard,
+  propControls: [],
+  // FIXME this form gets embedded within a form, and should not have its own independent create/save controls
+  advancedConfig: <AutomatedAnalysisConfigForm isSettingsForm={false}></AutomatedAnalysisConfigForm>,
```

Comment:
This needs a a wrapper around this form to add a switch to choose default or custom settings?

Reply:
That's part of what I meant about this component not being properly set up right now. There should be a switch to hide or reveal these controls, but they also need to be refactored so that they can be used separately/outside of a `<Form>`. The `Create` button probably shouldn't be there and the `Save Configuration` should probably just happen every time a change occurs rather than requiring the user to click a button so that they aren't having to save within the config screen and then also separately press `Finish` on the Wizard. Or, the `Finish` button from the wizard needs to be hooked into the `Save Configuration` action as well. Seems out of scope of this particular PR.

Reply:
Ahh right makes sense to me! Just a little odd looking at the template menu is set as required but wizard is allowed to finish. We will fix this later then.